multimodal/tests/unittests/others/test_object_detection.py

```diff
@@ -91,6 +91,28 @@ def test_mmdet_object_detection_inference_list_str_dict(checkpoint_name):
     assert len(pred) == 10  # test data size is 100
 
 
+@pytest.mark.parametrize(
+    "checkpoint_name",
+    [
+        "yolov3_mobilenetv2_320_300e_coco",
+    ],
+)
+def test_mmdet_object_detection_inference_xywh_output(checkpoint_name):
+    mmdet_image_name = download_sample_images()
+
+    predictor = MultiModalPredictor(
+        hyperparameters={
+            "model.mmdet_image.checkpoint_name": checkpoint_name,
+            "model.mmdet_image.output_bbox_format": "xywh",
```

Comment:
check functional equalvance of the two output formats, otherwise the test case would simply pass even without this hyperparameter.

Reply:
Good point, just updated the test